examples/basic/script.jl

```diff
@@ -225,7 +225,7 @@ mu = hcat(mu1, mu2)
 C = pairwise(SqEuclidean(), support'; dims=2)
 for λ1 in (0.25, 0.5, 0.75)
     λ2 = 1 - λ1
-    a = sinkhorn_barycenter(mu, C, 0.01, [λ1, λ2]; max_iter=1000)
+    a = sinkhorn_barycenter(mu, C, 0.01, [λ1, λ2], SinkhornBarycenterGibbs())
```

Comment:
Maybe it would make sense to just use `SinkhornGibbs` here? The function `sinkhorn_barycenter` already indicates that we want to compute the barycenter, it seems.

Reply:
Agreed!